CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Added
 
+* Certbot's official Docker images are now based on Alpine Linux 3.9 rather
```

Comment:
nit: Can you move this from the added section to the changelog to the changed section?

Reply:
Fixed